docs/installation_quickstart.rst

```diff
@@ -13,7 +13,7 @@ your environment is ready to receive traces.
 Installation
 ------------
 
-Install with :code:`pip`::
+Install with :code:`bash`::
```

Comment:
pip and bash? lol the snippet isn't bash specific - will also work with any unix-y shell

Reply:
not sure why it was "pip" before, if this is for code highlighting "pip" isn't a language

Reply:
oh wait it's the language directive lol...

Reply:
didn't even notice